src/models/FGAuxiliary.cpp

```diff
@@ -85,7 +85,6 @@ FGAuxiliary::FGAuxiliary(FGFDMExec* fdmex) : FGModel(fdmex)
   vAeroPQR.InitMatrix();
   vMachUVW.InitMatrix();
   vWindUVW.InitMatrix();
```

Comment:
You should remove the member `vWindUVW` as well since it has been added specifically for the Pitot angle computations (commit 32cacb4).